incubator/binding-mqtt.spec/src/main/resources/META-INF/zilla/mqtt.idl

```diff
@@ -77,7 +77,8 @@ scope mqtt
 
     struct MqttSessionState
     {
-        string16 clientId;
+        uint8 version = 1;
+        string16 clientId = null;
```

Comment:
Given that `clientId` is removed from all the `${mqtt:session()...}` expressions in the scripts, do we now need to remove this unused field in session state?